exo.core.component.organization.api/src/main/java/org/exoplatform/services/organization/AccountTemporaryLockedException.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2013 eXo Platform SAS.
```

Comment:
wrong header. we need copyright 2022 and AGPL license.

Reply:
For meeds, it is LGPL, right ? 
AGPL is for exoplatform.